manifests/host.pp

```diff
@@ -1,5 +1,48 @@
 # == Define: dhcp::host
```

Comment:
This is old rdoc style documentation. Would you mind changing this while you're at it?
```suggestion
# @summary Define a DHCP host
```